src/AWS.Deploy.Common/Recipes/OptionSettingItem.ValueOverride.cs

```diff
@@ -92,11 +92,11 @@ public object GetValue(IDictionary<string, string> replacementTokens, IDictionar
         /// Thrown if one or more <see cref="Validators"/> determine
         /// <paramref name="valueOverride"/> is not valid.
         /// </exception>
-        public void SetValue(IOptionSettingHandler optionSettingHandler, object valueOverride)
+        public void SetValue(IOptionSettingHandler optionSettingHandler, IValidatorFactory validatorFactory, object valueOverride)
```

Comment:
could you remove the `IValidatorFactory` as a dependency to `SetValue` and instead pass in the list of validators, which is the result of `validatorFactory.BuildValidators(this)`? `SetOptionSettingValue` in `OptionSettingHandler` is meant to handle anything related to services and only pass down simplified objects to `SetValue` in `OptionSettingItem`.

Reply:
Done